dev-perl/alien_build/alien_build-2.84.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="Build external dependencies for use in CPAN"
+DESCRIPTION="This module provides tools for building external (non-CPAN) dependencies for CPAN. \
+It is mainly designed to be used at install time of a CPAN client, and work closely with \
+Alien::Base which is used at runtime."
+HOMEPAGE="https://metacpan.org/pod/Alien::Build"
+COPYRIGHT="2011-2022 by Graham Ollis"
+LICENSE="Artistic"
+REVISION="1"
+SOURCE_URI="https://cpan.metacpan.org/authors/id/P/PL/PLICEASE/Alien-Build-$portVersion.tar.gz"
+CHECKSUM_SHA256="8e891fd3acbac39dd8fdc01376b9abff931e625be41e0910ca30ad59363b4477"
+SOURCE_DIR="Alien-Build-$portVersion"
+#PATCHES="alien_sdl-$portVersion.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	alien_build$secondaryArchSuffix = $portVersion
+	"
+# This package doesn't provide a suffix-less version on secondaryArch because it calls pkg-config
+# at runtime. Using it in an "any" recipe would call the wrong version on x86, so this deliberately
+# only supports architecture-specific downstream packages.
+REQUIRES="
+	haiku$secondaryArchSuffix
+	capture_tiny
+	ffi_checklib
+	file_chdir
+	file_which
+	path_tiny
+	vendor_perl
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	file_which
+	"
+BUILD_PREREQUIRES="
+	cmd:make
+	cmd:perl
+	"
+
+TEST_REQUIRES="
+	capture_tiny
+	ffi_checklib
+	file_chdir
+	path_tiny
```

Comment:
Should we maybe add `cmd:gcc$secondaryArchSuffix` here (and to BUILD_PREREQUIRES above)? Some test displays an error, but is still successful without it).